react/index.tsx

```diff
@@ -174,13 +201,14 @@ const RichText: FunctionComponent<Props> = ({
     marked(formatIOMessage({ id: text, intl })),
     sanitizerConfig
   )
+
   return (
     <div
       id={htmlId}
       className={`${generateBlockClass(
         styles.container,
         blockClass
-      )} flex ${alignToken} ${itemsToken} ${justifyToken} ${font} ${textColor}`}
+      )} flex ${alignToken} ${itemsToken} ${justifyToken} ${sanitizeFont(font)} ${textColor}`}
```

Comment:
how about the other ones, do they not come from user input?

Reply:
Sanitized color as well